packages/typespec-vs/src/VSExtension.cs

```diff
@@ -148,23 +152,21 @@ private void LogStderrMessage(string? message)
         }
 
 #if DEBUG
-        private static bool InDevelopmentMode()
-        {
-            return string.Equals(
-              Environment.GetEnvironmentVariable("TYPESPEC_DEVELOPMENT_MODE"),
-              "true",
-              StringComparison.OrdinalIgnoreCase);
+        private static bool InDevelopmentMode () {
```

Comment:
Some unexpected reformatting in this file, please revert. I think automated format in PR validation is not handling the `#if DEBUG`, but I would expect VS to use our .editorconfig, so not sure how this got reformatted for you.

Reply:
Updated with VS2022 format. Still resulted in a few changes. But those changes look good.